nasa/ow_simulator
# Oceanwater 1185 correct normal parameter in arm find surface

## Linked Issues:
Jira Ticket 🎟️ | [OCEANWATER-1185](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-1185)

## Summary of Changes
* The normal parameter in ArmFindSurface now represents the surface normal at the point being interrogated as the name implies and as the unification document specifies. Previously this parameter represented the anti-normal of the surface.

## Test
1. Run `rostest ow_sim_tests test_arm_find_surface.test`
2. Verify all test units pass
